dashboard/src/modules/components/SidebarComponent/sideMenuOptions.js

```diff
@@ -41,22 +27,8 @@ export const menuOptions = [
       },
     ],
   },
-  {
-    group: { key: "tools", title: "Tools" },
-    submenu: [
-      {
-        name: "Search",
-        submenu: true,
-        key: "search",
-        submenuOf: "tools",
-        link: APP_ROUTES.SEARCH,
-      },
-    ],
-  },
 ];
 
 export const menuOptionsNonLoggedIn = [
   { key: "dashboard", link: "/", name: "Dashboard" },
-  { key: "search", link: APP_ROUTES.SEARCH, name: "Search" },
-  { key: "expore", link: APP_ROUTES.EXPLORE, name: "Explore" },
 ];
```

Comment:
It is somewhat unsatisfying having different sidebars for logged-in vs. non-logged-in users.  That is, I _think_ we want to offer the "Results" page and the "Visualization" page to both logged-in and non-logged-in users; so, the only difference in the sidebar would be the "Overview", and we could handle that by having it detect the lack of login and offer a link to let the user log in "to see your own results".  And then the sidebar wouldn't need to be sensitive to login state (and we wouldn't have to maintain two sets of menu options).

Reply:
Yes: Profile and Overview make no sense if not logged in, and there may be some value in hiding them for simple "public dataset browsing". Since the "Profile" button doesn't exist except when logged in, "protecting" the page route doesn't really seem meaningful; while having "Overview" in the sidebar (which redirects you to login) makes it clear that exists. Right now, with "Search" and "Explore" removed, the sidebar will be just "Dashboard"; which interestingly is clickable, and takes you to the view which, when logged in, is labeled "Results". (While, when logged in, "Dashboard" is an unclickable title in a different font.)

So yes: having a *consistent* sidebar would be really nice, and especially where the same button takes you to the "default not-logged-in view" we normally call "Results". And we'd like to see Overview there (which takes you to log in) as well as "Visualization".

Reply:
> Since the "Profile" button doesn't exist except when logged in, "protecting" the page route doesn't really seem meaningful

Well, given that the user **could** type the URL into the browser's navigation bar, it's not silly to protect that route, and I'm OK with us protecting the _routes_ which require login (and with us **not** protecting routes which don't require login), but I'm thinking that the contents of the _sidebar_ don't need to be sensitive to that context.

>  the sidebar will be just "Dashboard"; which interestingly is clickable, and takes you to the view which, when logged in, is labeled "Results". (While, when logged in, "Dashboard" is an unclickable title in a different font.)

Yes, I noticed that, too, which was partly what prompted my comment, above.  Having a sidebar with just one clickable entry seems like overkill; having two, similar entries -- one clickable and one not -- seems odd; and, it just seems like having two separate sidebars isn't necessary when the two would do (or we would want them to do) substantially the same stuff.

Reply:
> Well, given that the user could type the URL into the browser's navigation bar,

Fair point, if any of that actually worked, although it doesn't. `.../profile` results in the 404 elk while `.../overview` redirects to the login page (even if already logged in) and then to the default Results page ...